### ISSUE TITLE
Built-in calendar?

### ISSUE THREAD

**Issue by RafeeDaBoy** [mhss1/MyBrain#41]
**Is your feature request related to a problem?**
Adding events in the app without having a calendar app installed is pretty good, but maybe having a built-in calendar would be even more nicer as there it'd then only require to have this app installed making it an all in one solution for general productivity stuff. It is bit of a big feature.

**Comment by mhss1** [comment]
This feature is already available in the new version. Make sure that you have updated the app to v1.0.1

**Comment by RafeeDaBoy** [comment]
Yes, adding calender events is present. What I meant was a calendar with week, month and year view.

**Comment by mhss1** [comment]
Added to roadmap 👍

**Comment by RafeeDaBoy** [comment]
Amazing, can't wait for it! I love all the amazing work you've done so far!

**Comment by Terkyz** [comment]
Hey, please also add compability to others calendars such Protonmail or Tutanota calendars!

**Comment by AliWhoi666** [comment]
> Hey, please also add compability to others calendars such Protonmail or Tutanota calendars!

I don't think it'll be possible how they encrypt their calenders. Would love to have a built-in calendar tho!

**Comment by abnvkdl** [comment]
Is there any progress on week, day, month view.
Mybrain has become more of a productivity app, and it would be nice if the calender section improved

**Comment by mhss1** [comment]
Hello, unfortunately this is not considered in the near future so will close it for now.
(Calendar monthly view is now supported though)
